wp-content/mu-plugins/register-taxonomies.php

```diff
@@ -74,7 +74,7 @@
    *
    * @author NYC Opportunity
    */
-  register_taxonomy('age', ['programs'], array(
+  register_taxonomy('agerangeserved', ['programs'], array(
```

Comment:
Can you keep this as `age`? It's more readable. You can discard all the changes except in archive.php and programs-archive.js